kratos/tests/test_cad_json_input.py

```diff
@@ -0,0 +1,20 @@
+﻿from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import os
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+
+class TestCadJsonInput(KratosUnittest.TestCase):
+
+    def test_cad_json_input_read(self):
+        cad_model = KratosMultiphysics.Model()
+        cad_model_part = cad_model.CreateModelPart("CadModelPart")
+
+        KratosMultiphysics.CadJsonInput(GetFilePath("auxiliar_files_for_python_unittest/cad_json_files/single_square")).ReadModelPart(cad_model_part)
+
```

Comment:
The test is not checking nothing!

Reply:
At least it checks that the input is read without errors. That's already something. As soon as we have passed all geometry/ nurbs/ model part entities to python I will add more here. However, I prefer to not mix up too many prs...

Reply:
Now that the mp is through, I added some tests However, there is not a lot to test so far...